runtime/parachains/src/inclusion.rs

```diff
@@ -413,15 +414,26 @@ impl<T: Trait> Module<T> {
 					Error::<T>::NotCollatorSigned,
 				);
 
-				check_cx.check_validation_outputs(
-					para_id,
-					&candidate.candidate.commitments.head_data,
-					&candidate.candidate.commitments.new_validation_code,
-					candidate.candidate.commitments.processed_downward_messages,
-					&candidate.candidate.commitments.upward_messages,
-					T::BlockNumber::from(candidate.candidate.commitments.hrmp_watermark),
-					&candidate.candidate.commitments.horizontal_messages,
-				)?;
+				if let Err(err) = check_cx
+					.check_validation_outputs(
+						para_id,
+						&candidate.candidate.commitments.head_data,
+						&candidate.candidate.commitments.new_validation_code,
+						candidate.candidate.commitments.processed_downward_messages,
+						&candidate.candidate.commitments.upward_messages,
+						T::BlockNumber::from(candidate.candidate.commitments.hrmp_watermark),
+						&candidate.candidate.commitments.horizontal_messages,
+					)
+				{
+					log::debug!(
```

Comment:
This would only be printed on native but also compiled for wasm :P

Reply:
oh, why is that? Is it because the logger is not initialized?

Reply:
Yeah

Reply:
That's a good catch, thanks. Is there a way to make it work on wasm? 

Or do you recommend to switch to only native logging? I was shooting to get logging in both native and wasm and the cost of the whole PR doesn't seem too big: the binary size of this PR compared to master is ~1000 bytes.

Reply:
I'm fine with logging from wasm.

We would need to initialize the `RuntimeLogger`. Like I had done it in the runtime api

Reply:
Ok, I can slap a call to `RuntimeLogger::init` just before logging.

But that feels such an ad-hoc place to initialize a logger. Ideally we do that only once, close to the entrypoint into the runtime. I know that initializing the logger twice won't harm, but that nonetheless feels non elegant. Something that we should think about